src/Datadog.Trace/Vendors/Serilog/Core/LevelOverrideMap.cs

```diff
@@ -66,11 +66,19 @@ public LevelOverride(string context, LoggingLevelSwitch levelSwitch)
                 .ToArray();
         }
 
-        public void GetEffectiveLevel(string context, out LogEventLevel minimumLevel, out LoggingLevelSwitch levelSwitch)
+        public void GetEffectiveLevel(
+#if FEATURE_SPAN
```

Comment:
We can enable this for netcore if we change up the flags with a transform. Thoughts?

Reply:
Done.

Reply:
You didn't manually add this to the file, right? You just found the ifdef and added it to our csproj?

Reply:
Correct.